From 6f6492b63181ad3cf6f98538870bb59923c6855c Mon Sep 17 00:00:00 2001
From: WeiguangTWK <weiguangtwk@outlook.com>
Date: Fri, 10 Jul 2026 13:53:49 +0800
Subject: [PATCH] Neko LineageOS: [Misc] Prevent new cloned clang-r416183b
 flooding repo status

To build LineageOS on 5.10 Kernel with Android KMI-stable, you also need to build it with clang-r416183b

This commit let git/repo not count clang-r416183b into untracked changes to make output of status better

Change-Id: Ib79c1bb76c8fdba99d0432f0c521f4b837c83fe7
---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index 23f6953a..7273893c 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,2 +1,3 @@
 **/python3/**/*.pyc
 **/python3/**/__pycache__
+clang-r416183b/
-- 
2.43.0

